python/spec/fixtures/requirements/jinja_requirements.txt

```diff
@@ -1,5 +0,0 @@
-{%- if cookiecutter.foo == "y" %}
```

Comment:
Are these not supported in latest pip anymore? What would happen if someone has these files checked in and working currently? Should raise a helpful error instead?

Reply:
It was never really supported, but pip offered a deprecated workaround that would let you pass a regex that it should ignore. That option has been removed here: https://github.com/pypa/pip/pull/8098

Reply:
It's pretty hard for us to differentiate between this and a regualr parse error, but I suppose we could catch the parse error and then check the file to see if it contains any of these jinja tags and raise a different error based on that? Might be a bit gnarly but could work

Reply:
Some more context here: https://github.com/dependabot/dependabot-core/pull/1537

Reply:
Ah yeah makes sense to drop this then. What error do we currently get if we try and parse this file? Seems fine as long as it raises the default `DependencyFileNotParseable`. Should we keep a spec that checks the exception so we can trace back to this and maybe link to the Jinja PR for ref?

Reply:
Yeah, I like that 👍 will put a test in